src/toil/options/__init__.py

```diff
@@ -0,0 +1,13 @@
+from typing import Optional
+
+from toil.lib.conversions import strtobool
+
+
+def convert_bool(b: str) -> bool:
+    """Convert a string representation of bool to bool"""
+    return bool(strtobool(b))
+
+
+def opt_strtobool(b: Optional[str]) -> Optional[bool]:
+    """Convert an optional string representation of bool to None or bool"""
+    return b if b is None else convert_bool(b)
```

Comment:
Maybe this should just live in `toil.lib.conversions` next to the non-`opt_` version `strtobool`? Than we could avoid adding this file.